components/noPodFoundError/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PodRootLoadError renders 1`] = `
+<DocumentFragment>
+  <div
+    class="PodBrowser-container"
+    data-testid="no-pod-message"
+  >
+    <h1>
+      Pod Not Found
+    </h1>
+    <p>
+      It appears you do not have a Pod.
```

Comment:
Can we make the link a button? if not, can we wrap the link in a p tag instead? and if not that, can we add a space at the end of "It appears you do not have a Pod. " so that there's a space in between the text and the link? Thx  
![image](https://user-images.githubusercontent.com/11729241/188927018-e2af4a5f-4fc6-4b4b-b250-0d31fff28419.png)

Reply:
When you say make a button, you mean make it look like button or actually using a button tag? If the former, classic PodBrowser purple button?

Reply:
Yeah make it look like a button, the classic purple one :D